lib/components/Checkbox/index.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import styled from 'styled-components';
+import theme from '../theme';
+
+const Checkbox = ({ className, checked, disabled, ...props }) => {
+  const cx = classNames(className);
+
+  return (
+    <CheckboxContainer className={cx} disabled={disabled}>
+      <HiddenCheckbox checked={checked} disabled={disabled} {...props} />
+      <StyledCheckbox checked={checked}>
+        <Icon viewBox="0 0 24 24">
+          <polyline points="20 6 9 17 4 12" />
```

Comment:
nit: these are coordinate pairs, right? When I looked at the example of `polyline` on MDN they're written with commas like this, which I think I prefer: `points="20,6 9,17 4,12"`

Reply:
Sure. I had copied that a while back and it worked, didn't spend the time to read MDN docs on it.